R/format.R

```diff
@@ -8,6 +8,59 @@
 #'
 #' @return Data frame with `SASformat` attributes for each variable.
 #'
+#' @section Format Checks: This function carries out a series of basic
+#'  checks to ensure the formats being applied make sense.
+#'
+#' 1) If the variable has a suffix of `DT`, `DTM`, `TM` (indicating a
+#'  numeric date/time variable) then a warning will be shown if there is
+#'   no format associated with it.
```

Comment:
The warning only fires when the user sets verbose = "warn" correct?

Reply:
No, that is not the case currently - will update so that this becomes the case.

Reply:
@bms63 - Would this also be the case for the 'message' also?

Reply:
NOTE TO SELF - need to update

- [ ] Function documentation
- [ ] Tests to include `verbose` option
- [ ] Additional test for a change in `verbose` option

Reply:
(Halfway through the tests - will continue tomorrow evening)

Reply:
> @bms63 - Would this also be the case for the 'message' also?

If the users sets to 'none` then nothing should come out of this function.  The other functions are set up like this.  It might be really good to make a test that uses `xportr_metadata()` and `xportr()` just to make sure these wrapper functions still work with `xportr_format()` update

Reply:
There is already a test in `test-xportr.R` which tests that `xportr()` works - is this sufficient do you think, or would you like another? (It passes with the updates to `xportr_format()` still).

Reply:
if a similar test already exists then no worries.  

maybe we should just have a `test-xportr-fcns-interactions.R` rather than my other proposed suggestion